containers/mit-scheme/README.md

```diff
@@ -0,0 +1,76 @@
+# Debian
+
+## Summary
+
+*Simple mit-scheme container with Git installed.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | Aisuko |
+| *Definition type* | Dockerfile |
+| *Published images* | mcr.microsoft.com/vscode/devcontainers/mit-scheme:11.1 |
+| *Available image variants* | 11.1 ([full list](https://mcr.microsoft.com/v2/vscode/devcontainers/mit-scheme/tags/list)) |
+| *Published image architecture(s)* | x86-64 |
+| *Works in Codespaces* | Yes |
+| *Container host OS support* | Linux, macOS, Windows |
+| *Container OS* | Debian |
+| *Languages, platforms* | MIT-Scheme |
+
+## Using this definition with an existing folder
+
+While the definition itself works unmodified, you can select the version of Debian the container uses by updating the `VARIANT` arg in the included `devcontainer.json` (and rebuilding if you've already created the container).
+
+```json
+"args": { "VARIANT": "11.1" }
+```
+
+You can also directly reference pre-built versions of `.devcontainer/base.Dockerfile` by using the `image` property in `.devcontainer/devcontainer.json` or updating the `FROM` statement in your own  `Dockerfile` to one of the following. An example `Dockerfile` is included in this repository.
```

Comment:
You'll need to update this to remove pre-build instructions as this image won't be prebuilt by our team.

Reply:
Done.